lib/matplotlib/_mathtext.py

```diff
@@ -1944,8 +1945,11 @@ class Parser:
     The grammar is based directly on that in TeX, though it cuts a few corners.
     """
 
-    _math_style_dict = dict(displaystyle=0, textstyle=1,
-                            scriptstyle=2, scriptscriptstyle=3)
+    class _MathStyle(enum.Enum):
+        DISPLAYSTYLE = enum.auto()
+        TEXTSTYLE = enum.auto()
+        SCRIPTSTYLE = enum.auto()
+        SCRIPTSCRIPTSTYLE = enum.auto()
 
```

Comment:
Where are these two used?

Reply:
Hmm, apparently nowhere. My guess is they were defined for compatibility with TeX, but never implemented. Looking at commit messages of #8151, I think that was the author's goal but maybe never got back to it.